test/src/module/info/manifestTest.c

```diff
@@ -530,6 +530,29 @@ testRun(void)
         varLstAdd(tablespace, varNewStrZ("tblspc1"));  // tablespaceName
         varLstAdd(tablespaceList, varNewVarLst(tablespace));
 
+        // Add more tablespaces to test bug
+        // for (unsigned int tsIdx = 3; tsIdx <= 4001; tsIdx++)
```

Comment:
Should we comment that this is removed but left here for a reason?

Reply:
I'm going to remove it completely. Perhaps we could add it to the performance test at some point but I don't think it is worth the time right now.

Reply:
Removed in ef221c6e23f57e22.